pkg/reloader/reloader.go

```diff
@@ -0,0 +1,178 @@
+package natsreloader
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
```

Comment:
Do you need to add this to the gopkg manifest?

Reply:
ok already added this one